### PR TITLE
README update

### PR DESCRIPTION
**Pull request by Dan2lle**

### CODE REVIEW

**Review by 0804EshitaSharma — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,67 +1,85 @@
...
 
-    f.  Email or text notifications on product purchase
+## Above and Beyond Functionality
 
-3.  **Stretch Requirements**
 
-    a.  Show user's recently viewed products.
+## Next Steps
+- We have successfully fulfilled nearly all project requirements, with the exception of the 'Show Recently Viewed Products' feature, which remains pending, and the partially completed 'Share Products with Friends' functionality. Completing these two goals would further enhance the app's overall functionality.
+- During our discussions, we explored additional potential features to enrich the shopping site's capabilities. These exciting prospects include multi-language support for broader accessibility, implementing an advanced product recommendation algorithm personalized to each user, as well as introducing user roles to create separate pages exclusively accessible to sellers and buyers, adding depth and sophistication to our platform.
```
Comment (0804EshitaSharma) [inline README.md:53]: Can we please have bullet points instead?

Comment (Dan2lle) [reply README.md:53]: Do you mean bullet points for each additional feature?

Comment (0804EshitaSharma) [reply README.md:53]: Yes

Comment (Dan2lle) [reply README.md]: Just updated it, let me know how that looks

Comment (0804EshitaSharma) [reply README.md]: It looks good, Thanks:)

**Pull request merged**